js/src/components/google-account-card/authorize-google-account-card.js

```diff
@@ -7,116 +7,60 @@ import { createInterpolateElement } from '@wordpress/element';
 /**
  * Internal dependencies
  */
-import { glaData } from '.~/constants';
 import AppButton from '.~/components/app-button';
-import AppDocumentationLink from '.~/components/app-documentation-link';
 import AccountCard, { APPEARANCE } from '.~/components/account-card';
-import useDispatchCoreNotices from '.~/hooks/useDispatchCoreNotices';
-import useGoogleAuthorization from '.~/hooks/useGoogleAuthorization';
+import readMoreLink from './read-more-link';
+import useGoogleConnectFlow from './use-google-connect-flow';
 import './authorize-google-account-card.scss';
 
-const readMoreLink = (
-	<AppDocumentationLink
-		context="setup-mc-accounts"
-		linkId="required-google-permissions"
-		href="https://docs.woocommerce.com/document/google-listings-and-ads/#required-google-permissions"
-	/>
-);
-
 /**
  * Renders an AccountCard based on Google appearance for requesting Google authorization from user.
  *
  * @param {Object} props React props.
  * @param {string} [props.additionalScopeEmail] Specify the email to be requested additional scopes. Set this prop only if wants to request a partial oauth to Google.
```

Comment:
`additionalScopeEmail` is changed to a required prop.

```suggestion
 * @param {string} props.additionalScopeEmail Specify the email to be requested additional permission scopes to Google.
```